test/uk/gov/hmrc/merchandiseinbaggage/smoketests/pages/PreviousDeclarationDetailsPage.scala

```diff
@@ -16,10 +16,21 @@
 
 package uk.gov.hmrc.merchandiseinbaggage.smoketests.pages
 import org.openqa.selenium.htmlunit.HtmlUnitDriver
+import org.scalatest.{Assertion, Suite}
+import uk.gov.hmrc.merchandiseinbaggage.smoketests.BaseUiSpec
+import uk.gov.hmrc.merchandiseinbaggage.smoketests.pages.PreviousDeclarationDetailsPage._
 
 object PreviousDeclarationDetailsPage extends Page {
   val path = "/declare-commercial-goods/previous-declaration-details"
-  val title = "Details of your existing declaration"
+  val title = "Details of your existing declaration - Declare commercial goods carried in accompanied baggage or small vehicles - GOV.UK"
```

Comment:
This look different from all the other smoke test `titles`. 
Why not have a test `pageTitle should startWith(title)` below

Reply:
looks like the existing titles doesn't seem to be in use....  we have been using the exact full titles  and equals check in the acceptance tests...
either way it's fine by me as long as it's consistant...